dist/index.html

```diff
@@ -1,15 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Weather API</title>
-</head>
-<body>
-    <h1>weather api</h1>
-
-    <script src="main.js"></script>
-
-</body>
+	<head>
+		<meta charset="UTF-8">
+		<meta http-equiv="X-UA-Compatible" content="IE=edge">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0,maximum-scale=1">
+		<title>Weather App</title>
+		<link href="http://fonts.googleapis.com/css?family=Roboto:300,400,700|" rel="stylesheet" type="text/css">
+		<link rel="stylesheet" href="./css/main.css">
+	</head>
+	<body>
+		<div class="site-content">
+			<div class="hero">
+				<div><h1>Weather App</h1></div>
+				<div class="container">
+					<form action="#" class="find-location">
+						<input id ="city" type="text" placeholder="Find your location...">
+						<input id="checkBtn" type="submit" value="Find">
+					</form>
+				</div>
+			</div>
+			<div class="forecast-table">
+				<img class="background" src="img/lenticular-cloud.jpg" alt="background-img">
+			</div>
+			
+			<footer class="site-footer">
+				<div class="container">
+					<div class="row">
+						<div class="col-md-3 col-md-offset-1">
+							<div class="social-links">
+								<a href="#"><i class="fa fa-facebook"></i></a>
+								<a href="#"><i class="fa fa-twitter"></i></a>
+								<a href="#"><i class="fa fa-google-plus"></i></a>
+								<a href="#"><i class="fa fa-pinterest"></i></a>
```

Comment:
The social icons do not show because we haven't added font awesome to the page yet.